wiki/People/The_Team/id.md

```diff
@@ -5,93 +5,94 @@ tags:
-| ![][flag_AU] [smoogipoo](https://osu.ppy.sh/users/1040328) | Pengembang osu!, Pecinta osu!mania, Penghilang kutu |
-| ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689) | Manajer Project Loved, Penasihat komunitas, Penjangkau Artis Unggulan |
-| ![][flag_ES] [Trosk-](https://osu.ppy.sh/users/3469385) | ART umum, Asisten turnamen |
-| ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416) | ART umum, Asisten turnamen, Pemelihara wiki |
+| ![][flag_GB] [-Mo-](https://osu.ppy.sh/users/2202163) | Penasihat bidang *mapping* dan *modding* |
+| ![][flag_US] [Chaos](https://osu.ppy.sh/users/2628870) | Penasihat bidang pengembangan dan moderasi komunitas |
+| ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335) | Koordinator umum, pengelola Featured Artist Project dan proyek-proyek komunitas lainnya, penjangkau Featured Artist |
+| ![][flag_JP] [flyte](https://osu.ppy.sh/users/3103765) | Pemimpin tim desain, pemerhati Creative Cloud |
+| ![][flag_GB] [JBHyperion](https://osu.ppy.sh/users/4879508) | Penasihat bidang *mapping* dan *modding* |
+| ![][flag_AR] [juankristal](https://osu.ppy.sh/users/443656) | Pengurus dan penyelenggara World Cup |
+| ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515) | Penasihat bidang *mapping* dan *modding* |
+| ![][flag_JP] [nanaya](https://osu.ppy.sh/users/2387883) | Pengembang situs web osu! (osu!web) |
+| ![][flag_PH] [Nathanael](https://osu.ppy.sh/users/2295078) | Pengasuh umum, penasihat bidang moderasi komunitas |
+| ![][flag_AU] [nekodex](https://osu.ppy.sh/users/102) | Pengembang situs web osu! (osu!web), [Featured Artist](https://osu.ppy.sh/beatmaps/artists/1) |
+| ![][flag_JP] [notbakaneko](https://osu.ppy.sh/users/10751776) | Pengembang situs web osu! |
+| ![][flag_DE] [Okoratu](https://osu.ppy.sh/users/1623405) | Penasihat bidang *mapping* dan *modding* |
+| ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) | Penasihat bidang *mapping* dan *modding*, pengelola Mappers' Guild, penjangkau Featured Artist |
+| ![][flag_MX] [Repflez](https://osu.ppy.sh/users/201392) | Pengasuh umum, pemelihara osu!wiki |
+| ![][flag_FR] [Shiro](https://osu.ppy.sh/users/113005) | Pengasuh umum |
+| ![][flag_AU] [smoogipoo](https://osu.ppy.sh/users/1040328) | Pengembang osu!, pecinta osu!mania, penumpas *bug* |
+| ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689) | Manajer Project Loved, penasihat bidang pengembangan komunitas, penjangkau Featured Artist |
+| ![][flag_ES] [Trosk-](https://osu.ppy.sh/users/3469385) | Pengasuh umum, penyelenggara turnamen |
+| ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416) | Pengasuh umum, penyelenggara turnamen, pemelihara osu!wiki |
 | ![][flag_AU] [Zallius](https://osu.ppy.sh/users/55) | Spesies yang terancam punah |
 
-## Grup pengguna
+Di samping nama-nama di atas, osu! juga memiliki [tim dukungan akun](Account_support_team) (*account support team*) yang bergerak secara terpisah untuk membantu menangani berbagai permasalahan yang bersifat pribadi.
```

Comment:
```suggestion
Di samping nama-nama di atas, osu! juga memiliki [account support team](Account_support_team) (*account support team*) yang bergerak secara terpisah untuk membantu menangani berbagai permasalahan yang bersifat pribadi.
```
i think it should be `account support team` as written in the wiki/id version (https://osu.ppy.sh/wiki/id/People/The_Team/Support_Team and https://osu.ppy.sh/wiki/id/People/The_Team/Account_support_team).

Reply:
ya that's why i added ``(*account support team*)`` next to the link for clarity

Reply:
Eh, sorry, i wasn't thorough when i checked it D: